docs/src/routes/docs/components/dot.svx

```diff
@@ -0,0 +1,86 @@
+---
+name: Dot
+---
+
+<script>
+  import { Dot, Button } from 'attractions';
+  import Showcase from 'src/containers/docs/showcase.svelte';
+  import ColorPreview from 'src/components/docs/color-preview.svelte';
+</script>
+
+# Dot
+
+A dot.
+
+<Showcase>
+  <div slot="showcase" class="padded">
+    <Button>
+      Plain ol' dot: <Dot />
+    </Button>
+    <Button>
+      New info <Dot info class="ml" />
+    </Button>
+    <Button>
+      <Dot success class="mr" />Mission accomplished
+    </Button>
+    <Button>
+      Take care! <Dot attention class="ml" />
+    </Button>
+    <Button>
+      Small warning <Dot danger small class="ml" />
+    </Button>
+  </div>
+  <div slot="source">
+
+```svelte
+<Button>
+  Plain ol' dot: <Dot />
+</Button>
+<Button>
+  New info <Dot info class="ml" />
+</Button>
+<Button>
+  <Dot success class="mr" />Mission accomplished
+</Button>
+<Button>
+  Take care! <Dot attention class="ml" />
+</Button>
+<Button>
+  Small warning <Dot danger small class="ml" />
+</Button>
+```
+
+  </div>
+</Showcase>
+
+
+## Properties {#properties}
+
+### Style Properties
+
+| Name | Default | Type | Description |
+| ---- | ------- | ---- | ----------- |
+| **`info`** | `false` | `boolean` | Applies the `$info` color to the dot |
+| **`attention`** | `false` | `boolean` | Applies the `$attention` color to the dot |
+| **`danger`** | `false` | `boolean` | Applies the `$danger` color to the dot |
+| **`success`** | `false` | `boolean` | Applies the `$success` color to the dot |
+| **`small`** | `false` | `boolean` | Decreases the size of the dot |
+
+### Class Properties
```

Comment:
Don't forget `restProps`. Yes, that will mean creating a _Functional properties_ heading just for this one entry, but consistency above all